scalene/replacement_fork.py

```diff
@@ -0,0 +1,14 @@
+import os
+from scalene.scalene_profiler import Scalene
+import signal
+
+@Scalene.shim
+def replacement_fork(scalene: Scalene):
+    orig_fork = os.fork
+
+    def fork_replacement():
+        result = orig_fork()
+        if result == 0:
+            signal.raise_signal(signal.SIGTSTP)
```

Comment:
Annoying to have the signal (SIGTSTP) here and in the other file - would be better if it were in only one place.